bitcoin/src/taproot/serialized_signature.rs

```diff
@@ -49,7 +49,7 @@ impl PartialEq<SerializedSignature> for [u8] {
 
 impl PartialOrd for SerializedSignature {
     fn partial_cmp(&self, other: &SerializedSignature) -> Option<core::cmp::Ordering> {
-        Some((**self).cmp(&**other))
+        Some((self).cmp(other))
```

Comment:
In b1e029fa85ef045fa3adc5ff83226ae536907054:

Parens around `self` are not necessary.

Reply:
Done